Blazor_Foundation_6_Test_Shared/Documentation/Containers/MediaObject.razor

```diff
@@ -1,6 +1,146 @@
+                <p>What is the most resilient parasite? Bacteria? A virus? An intestinal worm? An idea. Resilient... highly contagious. Once an idea has taken hold of the brain it's almost impossible to eradicate. An idea that is fully formed - fully understood - that sticks; right in there somewhere.</p>
+            </MediaObjectSection>
+            <MediaObjectSection Class="bottom">
+                <OpenCodeDev.Blazor.Foundation.Documentation.Media.Thumbnail>
+                    <img src="https://get.foundation/sites/docs/assets/img/media-object/avatar-3.jpg" />
+                </OpenCodeDev.Blazor.Foundation.Documentation.Media.Thumbnail>
+            </MediaObjectSection>
+        </MediaObjectWrapper>
+        <p>
+            In flexbox mode, you can use the >flexbox helper classes instead to get the same result. The <code>.align-*</code> classes
+            can be used on the container to align every child section at once, or individual child sections can be aligned with
+            <code>.align-self-*</code> classes.
+        </p>
+
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderTitle="Razor" HeaderBGcolor="#5d2b90"
+                     Content="PE1lZGlhT2JqZWN0V3JhcHBlcj4KICAgIDxNZWRpYU9iamVjdFNlY3Rpb24gQ2xhc3M9ImFsaWduLXNlbGYtbWlkZGxlIj4KICAgICAgICA8VGh1bWJuYWlsPgogICAgICAgICAgICA8aW1nIHNyYz0iYXNzZXRzL2ltZy9tZWRpYS1vYmplY3QvYXZhdGFyLTIuanBnIiAvPgogICAgICAgIDwvVGh1bWJuYWlsPgogICAgPC9NZWRpYU9iamVjdFNlY3Rpb24+CiAgICA8TWVkaWFPYmplY3RTZWN0aW9uIENsYXNzPSJtYWluLXNlY3Rpb24iPgogICAgICAgIDxoND5XaHkgaXMgaXQgc28gaW1wb3J0YW50IHRvIGRyZWFtPzwvaDQ+CiAgICAgICAgPHA+U28sIG9uY2Ugd2UndmUgbWFkZSB0aGUgcGxhbnQsIGhvdyBkbyB3ZSBnbyBvdXQ/IEhvcGUgeW91IGhhdmUgc29tZXRoaW5nIG1vcmUgZWxlZ2FudCBpbiBtaW5kIHRoYW4gc2hvb3RpbmcgbWUgaW4gdGhlIGhlYWQ/IEEga2ljay4gV2hhdCdzIGEga2ljaz8gVGhpcywgQXJpYWRuZSwgd291bGQgYmUgYSBraWNrLjwvcD4KICAgICAgICA8cD5XaGF0IGlzIHRoZSBtb3N0IHJlc2lsaWVudCBwYXJhc2l0ZT8gQmFjdGVyaWE/IEEgdmlydXM/IEFuIGludGVzdGluYWwgd29ybT8gQW4gaWRlYS4gUmVzaWxpZW50Li4uIGhpZ2hseSBjb250YWdpb3VzLiBPbmNlIGFuIGlkZWEgaGFzIHRha2VuIGhvbGQgb2YgdGhlIGJyYWluIGl0J3MgYWxtb3N0IGltcG9zc2libGUgdG8gZXJhZGljYXRlLiBBbiBpZGVhIHRoYXQgaXMgZnVsbHkgZm9ybWVkIC0gZnVsbHkgdW5kZXJzdG9vZCAtIHRoYXQgc3RpY2tzOyByaWdodCBpbiB0aGVyZSBzb21ld2hlcmUuPC9wPgogICAgPC9NZWRpYU9iamVjdFNlY3Rpb24+CiAgICA8TWVkaWFPYmplY3RTZWN0aW9uIENsYXNzPSJhbGlnbi1zZWxmLWJvdHRvbSI+CiAgICAgICAgPFRodW1ibmFpbD4KICAgICAgICAgICAgPGltZyBzcmM9ImFzc2V0cy9pbWcvbWVkaWEtb2JqZWN0L2F2YXRhci0zLmpwZyIgLz4KICAgICAgICA8L1RodW1ibmFpbD4KICAgIDwvTWVkaWFPYmplY3RTZWN0aW9uPgo8L01lZGlhT2JqZWN0V3JhcHBlcj4=" />
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Stack on Small</h2>
+        <p>
+            By adding the <code>.stack-for-small</code> class, you can make your media object responsive. Images will get a width of 100%,
```

Comment:
dont add . before class except for native html tags told ya b4

Reply:
i have made the changes